src/fal/run_scripts.py

```diff
@@ -96,6 +108,42 @@ def ref_resolver(target_model_name: str, target_package_name: Optional[str] = No
     return ref_resolver
 
 
+def _get_firestore_writer(model: ParsedModelNode, manifest: Manifest):
+    # Use the application default credentials
+    cred = credentials.ApplicationDefault()
+    app_name = str(uuid.uuid4())
+
+    app = firebase_admin.initialize_app(cred, {
+        'projectId': model.database,
+    }, name=app_name)
+
+    db = firestore.client(app=app)
+
+    def _dict_to_document(data: Dict, key_column: str):
```

Comment:
This could be outside of the function definition, right?

Reply:
I prefer it stays within the scope of this function